.docs/content/1.installation/2.windows/2.wsl2-with-genie.md

```diff
@@ -0,0 +1,100 @@
+# WSL2 with genie
+
+::alert{type="warning"}
+This installation method is not recommended for production environments.
+::
+
+This installation will use [`genie`](https://github.com/arkane-systems/genie). Genie is a tool which gives you a way to run systemd as pid 1. The usage of this tool will disable some Windows goodies within your Ubuntu distribution.
+
+::alert{type="warning"}
+We does not recommend to use this installation method, even for development environment since WSL2 support natively `systemd`. We recommend to use [WSL2 with systemd](./1.wsl2-with-systemd.md).
+::
+
+::alert{type="info"}
+You must have installed WSL2 and Ubuntu before continuing. If you haven't, please refer to the [WSL2 installation](./0.installation-using-wsl2.md) method.
+::
+
+## Install genie
+
+To install `genie`:
+
+```bash
+cd /tmp
+wget --content-disposition "https://gist.githubusercontent.com/djfdyuruiry/6720faa3f9fc59bfdf6284ee1f41f950/raw/952347f805045ba0e6ef7868b18f4a9a8dd2e47a/install-sg.sh"
+chmod +x install-sg.sh
+./install-sg.sh
+rm install-sg.sh
+```
+
+::alert{type="info"}
+Please, read the [script `install-sg.sh`](https://gist.githubusercontent.com/djfdyuruiry/6720faa3f9fc59bfdf6284ee1f41f950/raw/952347f805045ba0e6ef7868b18f4a9a8dd2e47a/install-sg.sh) before executing it.
+::
+
+Finally, in order to avoid some issues, you need to mask some services.
+
+```bash
+sudo systemctl disable getty@tty1.service multipathd.service multipathd.socket ssh.service
+sudo systemctl mask systemd-remount-fs.service
+```
+
+::alert{type="info"}
+You can find more information about this issue on the [genie documentation](https://github.com/arkane-systems/genie#warning-timing-out).
+::
+
+Then, you need to exit your Ubuntu distribution and restart it.
+
+```bash
+exit
+```
+
+```powershell
+wsl --shutdown
+```
+
+::alert{type="warning"}
+Please, does not start a WSL2 session with native `systemd` in parallel of `genie`. This will cause some issues with k3s when you will try to install it.
```

Comment:
```suggestion
Please, do not start a WSL2 session with native `systemd` in parallel of `genie`. This will cause some issues with k3s when you will try to install it.
```